style2.css

```diff
@@ -15,6 +15,17 @@
   margin: 50px 28px;
 }
 
+#popup-demo > a {
+  flex-grow: 1;
+  padding: 10px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  display: flex;
```

Comment:
- [ ] Kindly make sure your details pop-up design is as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/179846825-b9315699-fe52-49c7-91f2-df9b8bf0e18a.png)  |  ![image](https://user-images.githubusercontent.com/74814780/179846846-b86f5ea2-86ad-4c5c-b59b-0b9312b0b86f.png)